core/src/main/java/org/apache/iceberg/actions/RewritePositionDeletesCommitManager.java

```diff
@@ -55,12 +55,13 @@ public void commit(Set<RewritePositionDeletesGroup> fileGroups) {
     RewriteFiles rewriteFiles = table.newRewrite().validateFromSnapshot(startingSnapshotId);
 
     for (RewritePositionDeletesGroup group : fileGroups) {
+      long maxSequenceNumber = group.maxRewrittenDataSequenceNumber();
```

Comment:
What about either calling this `maxRewrittenDataSequenceNumber` and adding an empty line before the for loop below or using `group.maxRewrittenDataSequenceNumber()` directly?

Reply:
Done, use the call directly